manifest-generation/runtime-aci-tile.yml

```diff
@@ -0,0 +1,412 @@
+  - reference: ".properties.apic_vmm_controller" 
+    label: Apic Vmm Controller
+    description: date_type:string 
+  - reference: ".properties.apic_name_prefix" 
+    label: Apic name prefix
+    description: date_type:string 
+  - reference: ".properties.apic_policy_tenant" 
+    label: Apic Policy Tenant
+    description: date_type:string 
+  - reference: ".properties.apic_policy_default_epg" 
+    label: Apic Policy Default EPG
+    description: date_type:string 
+  - reference: ".properties.apic_policy_vrf_tenant" 
+    label: Apic Policy VRF Tenant
+    description: date_type:string 
+  - reference: ".properties.apic_policy_vrf" 
+    label: Apic Policy VRF
+    description: date_type:string 
+  - reference: ".properties.apic_l3out" 
+    label: Apic l3out
+    description: date_type:string 
+  - reference: ".properties.apic_ext_net" 
+    label: Apic external network
+    description: date_type:string_list 
+  - reference: ".properties.apic_service_phys_domain" 
+    label: Apic Service Physdomain
+    description: date_type:string 
+  - reference: ".properties.apic_encap_type" 
+    label: Apic Vmm encap type
+    description: date_type:string 
+  - reference: ".properties.apic_service_encap" 
+    label: Apic service encap
+    description: date_type:string 
+  - reference: ".properties.networking_aci_api_port" 
+    label: Networking ACI Api Port
+    description: date_type:port
+  - reference: ".properties.network_subnet" 
+    label: Network Subnet
+    description: date_type:string 
+  - reference: ".properties.network_ip_pool_start" 
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_ip_pool_end" 
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_app_dynamic_ext_ip_pool_start" 
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_app_dynamic_ext_ip_pool_start" 
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_app_static_ext_ip_pool_start"
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_app_static_ext_ip_pool_end" 
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_ip_pool_start"
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_ip_pool_end" 
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_subnets" 
+    label: Network node service subnets
+    description: date_type:string
+
+runtime_configs:
+- name: aci-containers-tile 
+  runtime_config: |
+    releases: 
+    - name: aci-containers
+      version: '0.0.9'
+    addons:
+    - name: network-setup-addon 
```

Comment:
I believe there are changes to job properties since version 0.0.9. Please compare the current job specs and cf_ops.yml to ensure that all necessary job properties are being specified.